src/Cache/MongoLock.php

```diff
@@ -107,6 +107,17 @@ public function forceRelease(): void
         ]);
     }
 
+    /** Creates a TTL index that automatically deletes expired objects. */
+    public function createTTLIndex(): void
+    {
+        $this->collection->createIndex(
+            // UTCDateTime field that holds the expiration date
+            ['expiration' => 1],
```

Comment:
If you have the flexibility to customize the field name, I'd suggest something like `expiresAt` for consistency with `createdAt` and `updatedAt`.

I didn't bother to raise this in the previous PR but it seems more relevant if there are also indexes involved and people may have eyes on the schema.

Reply:
[Laravel uses underscores](https://laravel.com/docs/11.x/eloquent#timestamps) for `created_at` and `updated_at`, I've updated to `expires_at`.